docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+    examples: |
+      - what's the conversation id?
+      - id
+      - What is the ID of this conversation?
+      - How do I send a POST request to this conversation?
+```
+
+:::note
+It is recommended
+to name intents that come from other devices with the `EXTERNAL_` prefix because it makes it
+easier to see which intents are expected to come from external devices when working with your training data.
+:::
+
+To tell the assistant which plant needs watering, you can define an entity that you'll post along with the intent.
+To be able to use the entity value directly in a response, define a slot by the same name:
+
+```rasa-yaml
+entities:
+  - plant
+
+slots:
+  plant:
+    type: unfeaturized
+```
+
+#### Rules and Actions
+
+You'll need two rules for the dry plant example: one to get the conversation ID back, and one
+that tells your assistant how to respond when it receives a message from the Raspberry Pi.
+
+```rasa-yaml
+- rule: conversation id
+  steps:
+  - intent: ask_id
+  - action: action_tell_id
+rules:
+  - rule: warn about dry plant
+    steps:
+    - intent: EXTERNAL_dry_plant
+    - action: utter_warn_dry
+```
+
+You'll need to add your custom action for telling the conversation ID to the domain too:
+
+```rasa-yaml
+actions:
+  - action_tell_id
+```
+
+#### Responses
+You'll need to define the response text for `utter_warn_dry`:
+
+```rasa-yaml
+responses:
+  utter_warn_dry: Your {plant} needs some water!
+```
+
+The response will use the value from the slot `plant` to warn about the specific plant that needs watering.
+
+### Try it out
+
+To try out the dry plant notification example, you'll need to start either [Rasa X](https://rasa.com/docs/rasa-x/)
+or a [`CallbackChannel`](your-own-website.mdx#callbackchannel).
+
+:::caution
+External Events and Reminders don't work in request-response channels like the `rest` channel or `rasa shell`.
+Custom connectors for assistants implementing reminders or external events should be built
+off of the [`CallbackInput` channel](your-own-website.mdx#callbackinput) instead of the `RestInput` channel.
+
+See the [reminderbot README](https://github.com/RasaHQ/rasa/blob/master/examples/reminderbot/README.md)
+for instructions on how to test your reminders locally.
+:::
+
+You'll also need to start the action server, since you need a custom action to tell you your conversation ID.
+
+Once you have retrained your model with the data above, you can try it out by
+getting the conversation ID back from the assistant:
+
+```yaml
+User: /ask_id
+  Bot: The ID of this conversation is user1234.
+      Trigger an intent with
+      curl -H "Content-Type: application/json" -X POST -d
+      '{"name": "EXTERNAL_dry_plant", "entities": {"plant": "Orchid"}}'
+      "http://localhost:5005/conversations/user1234/trigger_intent?output_channel=latest"
+```
+
+By running the returned POST request,
+you should see the bot respond in your channel:
+
+```yaml
+  Bot: Your Orchid needs some water!
+```
+
```

Comment:
i really don't get the need for this whole `/ask_id` use case :D maybe this is something to revisit in future as well